### ISSUE TITLE
Excepted <Entry point 'quantumespresso.pw' not found in group 'aiida.calculations'>

### ISSUE THREAD

**Issue by hongyi-zhao** [aiidateam/aiida-quantumespresso#501]
Hi,

I try to run the example given [here](https://aiida-tutorials.readthedocs.io/en/latest/_downloads/41fc098a6347cd8d1653a71b79cc51eb/launch_auto-wannier_workflow.py).

The python virtualenv is created with the following steps which ensures to make a exactly identical environment with the one shipped in [this Quantum Mobile's version](https://github.com/marvel-nccr/quantum-mobile/releases/tag/20.03.1):

```
1. In Quantum Mobile:
$ workon aiida
$ pip freeze > requirements.txt
2. On the real Ubuntu 19.10 host:
$ pyenv shell 3.6.9
$ pyenv virtualenv test
$ pyenv shell test
$ pip install --upgrade pip
$ pip install --no-deps -v -r requirements.txt
3. Run the above mentioned example:
 $ python launch_auto-wannier_workflow.py GaAs.xsf 
launched WorkChain<275> for structure AsGa
Use `verdi process list` or `verdi process show 275` to check the progress
$ verdi process show 275
Property     Value
-----------  -----------------------------------------------------------------------------------
type         Wannier90BandsWorkChain
state        Excepted <Entry point 'quantumespresso.pw' not found in group 'aiida.calculations'>


```

But I can find the quantumespresso.pw with following command:
```
$ verdi plugin list aiida.calculations | egrep quantumespresso.pw$
* quantumespresso.pw
```

Then I retried with aiida-quantum-mobile as the virtualenv name and the problem solved.  But I still cannot figure out the reason.

Any hints for this problem?

Regards

**Comment by sphuber** [comment]
Is `aiida-quantumespresso` installed? Is it listed if you run `pip freeze`?
If not install it. If it _is_ installed make sure to run `reentry scan`. It should then show up in `verdi plugin list aiida.calculations`. Make sure to restart the daemon before trying to resubmit

**Comment by hongyi-zhao** [comment]
@sphuber Thanks a lot for your notes.  

I've figured out the reason for the problem I mentioned above.  It's caused by the requirements.txt.  In detail, the file consists the following line:

`-e git+https://github.com/aiidateam/aiida-core.git@20fa6c252912d8af04d9926f93a9197d3cc2b651#egg=aiida_core`

And I firstly feed this file to `conda create `, but it cannot resolve the above line and I commented it out.  Afterwards, I retry this file through ` pip install` with the above line commented out.  As a result, the  aiida-core not installed, so I use `pip install  ` again to install this package.  This maybe the reason which created a inconsistent virtualenv for me.

When I uncomment the aiida-core line in the  requirements.txt file and use the` pip install --no-deps -r requirements.txt`  command to re-create the virtualenv, the problem solved.

Regards
